optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -3526,6 +3526,10 @@ def preprocess_inputs(
 
         text_prompt = processor.apply_chat_template(conversation, add_generation_prompt=True, tokenize=False)
 
+        # switch off add_bos_token if chat template already includes it
+        if "bos_token" in processor.tokenizer.chat_template:
+            processor.tokenizer.add_bos_token = False
+
         inputs = processor(images=image, text=text_prompt, videos=video, return_tensors="pt")
```

Comment:
I think we should set `processor.tokenizer.add_bos_token` back to its original value after `processor()` call in order not to modify the provided processor instance. What do you think?